src/etc/inc/interfaces.inc

```diff
@@ -2786,10 +2786,28 @@ REQUEST)
         ARGS="\${ARGS} -a \${NAMESERVER}"
     done
     /usr/local/sbin/ifctl -i ${wanif} -6nd \${ARGS}
-    /usr/local/sbin/ifctl -i ${wanif} -6sd \${new_domain_name:+"-a \${new_domain_name}"}
+    ARGS=
+    for DOMAIN in \${new_domain_name}; do
+        ARGS="\${ARGS} -a \${DOMAIN}"
+    done
+    /usr/local/sbin/ifctl -i ${wanif} -6sd \${ARGS}
     /usr/local/sbin/ifctl -i ${wanif} -6pd \${PDINFO:+"-a \${PDINFO}"}
     /usr/local/sbin/configctl -d interface newipv6 {$wanif}
     ;;
+INFOREQ)
```

Comment:
Nice. As far as I can see this was never once implemented and also absent from pfSense.

Reply:
Checking $REASON was introduced in https://github.com/opnsense/core/commit/d36f0f4f62557eb7caa150c7bd1611c08667990f. Not sure if it might have worked before.

Reply:
INFOREQ seems to be emitted when "information-only" is used so that REQUEST isn't called so that seems to be correct here indeed.